test/module/Entra/Get-EntraAuthorizationPolicy.Tests.ps1

```diff
@@ -0,0 +1,89 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+BeforeAll {  
+    if ((Get-Module -Name Microsoft.Graph.Entra) -eq $null) {
+        Import-Module Microsoft.Graph.Entra      
+    }
+    Import-Module (Join-Path $psscriptroot "..\Common-Functions.ps1") -Force
+    
+    $scriptblock = {
+        return @(
+            [PSCustomObject]@{
+                "DefaultUserRolePermissions"                = @{AllowedToCreateApps = "False"; AllowedToCreateSecurityGroups = "False"; AllowedToCreateTenants = "True";
+                    AllowedToReadBitlockerKeysForOwnedDevice = "True"; AllowedToReadOtherUsers = "False"; PermissionGrantPoliciesAssigned = "";
+                    AdditionalProperties = ""
+                }
+                "DeletedDateTime"                           = $null
+                "GuestUserRoleId"                           = "00aa00aa-bb11-cc22-dd33-44ee44ee44ee"
+                "DisplayName"                               = "AuthorizationPolicy"
+                "Description"                               = "AuthorizationPolicy"
+                "AllowEmailVerifiedUsersToJoinOrganization" = $True
+                "AllowedToSignUpEmailBasedSubscriptions"    = $True
+                "AllowInvitesFrom"                          = "everyone"
+                "AllowUserConsentForRiskyApps"              = ""
+                "AllowedToUseSspr"                          = $True
+                "BlockMsolPowerShell"                       = $True
+                "Id"                                        = "authorizationPolicy"
+                "Parameters"                                = $args
+            }
+        )
+    }
+
+    Mock -CommandName Get-MgPolicyAuthorizationPolicy -MockWith $scriptblock -ModuleName Microsoft.Graph.Entra
+}
+  
+Describe "Get-EntraAuthorizationPolicy" {
```

Comment:
Add test for parameter `Id`

Reply:
@KenitoInc Id parameter is missing in mg graph so it is not working on the Entra cmdlet.